components/Layout/Layout.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from "react";
+import Image from "next/image";
+
+import { Navbar } from "../Navbar";
+
+import styles from "./layout.module.scss";
+
+interface ILayoutProps {
+  children: ReactNode;
+  isHome?: boolean;
+}
+
+export const Layout = ({ children, isHome }: ILayoutProps) => {
```

Comment:
What do you think about exporting it as default? And then in the `index.ts` you just do `export {default} from './Layout'`?